packages/dcos-diagnostics/extra/dcos-diagnostics-master.service

```diff
@@ -13,5 +13,6 @@ User=dcos_diagnostics
 # Allow r/w access to the socket file
 SupplementaryGroups=dcos_adminrouter
 ExecStartPre=/opt/mesosphere/bin/bootstrap dcos-diagnostics-master
+ExecStartPre=/opt/mesosphere/bin/bootstrap dcos-checks-master
```

Comment:
Bootstrapping the dcos-checks service accounts as part of the `dcos-diagnostics` units feels a little hacky, since it's only tangentially related to that service, and these service accounts will be used regardless of whether `dcos-diagnostics` is running. I think this'll work, but maybe we want to consider adding a separate oneshot unit just for bootstrapping the checks service accounts, in order to decouple these things. A comment here noting what the service account is for would be helpful in any case.

Since this introduces a race (we might be running checks during installation postflight before this unit has had a chance to run), we should test that `dcos-diagnostics check` fails correctly (returns valid check output, no side effects, etc.) when the checks service account hasn't been bootstrapped yet.